src/dcap.rs

```diff
@@ -1,14 +1,15 @@
 use std::prelude::v1::*;
 
 use crypto::Secp256k1PrivateKey;
+use jsonrpc::RpcClient;
 use sgx_dcap_ql_rs::{sgx_qe_get_target_info, sgx_target_info_t};
 use sgxlib::sgx_types::{sgx_report_data_t, sgx_status_t};
 
 use crate::submit_dcap_quote;
 use crate::{ExecutionClient, RaFfi, SgxQuote, SgxReport};
 use eth_types::HexBytes;
 
-pub fn dcap_generate_quote(data: [u8; 64]) -> Result<SgxQuote, String> {
+pub fn dcap_generate_quote(arg: Option<[u8; 64]>) -> Result<SgxQuote, String> {
```

Comment:
the sgx-prover has already used this function, and I think changing it to Option is not necessary.